po/de.po

```diff
@@ -83,19 +83,19 @@ msgstr "Admin-Realm"
 
 #: privacyidea/static/components/audit/views/audit.stats.html:65
 msgid "Administrative actions"
-msgstr "Administrative Requests"
+msgstr "Administrative Aktionen"
 
 #: privacyidea/static/components/config/views/config.caconnectors.html:8
 msgid "All CA Connectors"
-msgstr "All CA Konnektoren"
+msgstr "Alle CA-Konnektoren"
 
 #: privacyidea/static/components/config/views/config.events.html:7
 msgid "All Event Handlers"
-msgstr "Alle Ereignisse"
+msgstr "Alle Ereignishandler"
 
 #: privacyidea/static/components/config/views/config.machineresolvers.html:7
 msgid "All Machine Resolvers"
-msgstr "Alle Maschinen Resolver"
+msgstr "Alle Maschinenauflöser"
```

Comment:
Please leave all occurences of "resolver" as resolver. No "Auflöser". One day we might rename this to "usersource".